### PR TITLE
docs: translation generation in README.md

### LINKED ISSUE #259: docs: Missing l10n documentation in the flutter project's `readme.md`
> **Note**: This [issue](https://github.com/VeryGoodOpenSource/very_good_cli/issues/691) was originally opened at [Very Good Cli](https://github.com/VeryGoodOpenSource/very_good_cli) by @tanmoy27112000.

## Description
The command to generate the new Localizations class when new strings are added to the arb file is missing.

## Steps To Reproduce

1. Create a flutter project using the cli
2. Check the `README.md` file

**Expected Behavior:**
The documentation should contain the command that has to be used to generate the classes for locale.

```sh
flutter gen-l10n
```

### PR DESCRIPTION
**Pull request by jsgalarraga**
## Description

Added documentation on how to generate translation files on README.md

Due to lack of activity in #275 I'm opening this PR addressing those changes
Fixes #259 
Closes #275 

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Review by alestiago — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -159,6 +159,15 @@ Update the `CFBundleLocalizations` array in the `Info.plist` at `ios/Runner/Info
 }
 ```
 
+### Generating Translations
+
+To use the latest translations changes, you will need to generate them beforehand. Flutter will automatically generate them per `flutter run`. Alternatively, if you would like to avoid running the entire application, you may do:
```
[the remark this answers is not in the document]
Comment (jsgalarraga) [reply README.md:164]: I agree that the sentence is confusing, let me rephrase it to something more clear. 
However, we already state that we use the flutter l10n stuff at the start of this translations section, see [here](https://github.com/VeryGoodOpenSource/very_good_core/blob/main/README.md?plain=1#L65), so I think there is not added value in repeating it here.

**Review by scarletteliza — APPROVED** [review]
